database/database.go

```diff
@@ -486,3 +488,29 @@ func (s *DatabaseService) DeleteExecutionPayloads(idFirst, idLast uint64) error
 	_, err := s.DB.Exec(query, idFirst, idLast)
 	return err
 }
+
+func (s *DatabaseService) SaveValidatorRefund(bidTrace *common.BidTraceV2, signedBlindedBeaconBlock *types.SignedBlindedBeaconBlock) error {
+	_signedBlindedBeaconBlock, err := json.Marshal(signedBlindedBeaconBlock)
+	if err != nil {
+		return err
+	}
+
+	validatorRefundEntry := ValidatorRefundEntry{
+		SignedBlindedBeaconBlock: NewNullString(string(_signedBlindedBeaconBlock)),
+
+		Slot:  bidTrace.Slot,
+		Epoch: bidTrace.Slot / uint64(common.SlotsPerEpoch),
+
+		BuilderPubkey:  bidTrace.BuilderPubkey.String(),
+		ProposerPubkey: bidTrace.ProposerPubkey.String(),
+
+		Value: bidTrace.Value.String(),
```

Comment:
We may need to store the bid fee recipient here, both as a validity condition but also to know which address needs to be refunded. We may also need to store the gas limit requested by the proposer. Even better may be to store the signed message by the proposer specifying the fee recipient and gas limit, so that we can cryptographic material as "proof".

Reply:
good point! i added a signedValidatorRegistration column to the table. i also added the fee recipient and gas limit as columns too to match the style of other tables.